app/views/travel_advice/_first_part.html.erb

```diff
@@ -1,4 +1,14 @@
-<%= render 'govuk_publishing_components/components/metadata', content_item.metadata %>
+<%
+  latest_update = simple_format(@travel_advice_presenter.latest_update, { class: "metadata__update" }, wrapper_tag: "span") if @travel_advice_presenter.latest_update.present?
```

Comment:
I'm mildly suspicious about this, because it feels like a slightly complex bit of code to have in the partial where we don't have coverage info, but I think it's fine.